models/users.js

```diff
@@ -829,6 +829,26 @@ const fetchUsersListForMultipleValues = async (documentKey, valueList) => {
   }
 };
 
+const updateUsernameToLowercase = async () => {
+  try {
+    const allUsers = await fetchAllUsers();
```

Comment:
the updateUsernameToLowercase() function is not returning a value. This means that the caller of the function will not know how many users were successfully updated.